extensions/target-specific/stl-helpers/stl-helpers.js

```diff
@@ -260,7 +260,27 @@ DbgObject.AddTypeDescription(
 );
 
 DbgObject.AddTypeDescription(
-    (type) => type.name().match(/^std::pair<.*>$/) != null,
+    (type) => type.name().match(/^std::__Cr::basic_string<.*>$/) != null,
```

Comment:
Is "std::__Cr" the only STL prefix that needs to be supported? What about "std::__1"? @KurtCattiSchmidt noticed that the AXTree extension was broken because some of the STL types being used have changed to "std::__1".  From looking online, it seems that "__1" is an inline namespace: https://stackoverflow.com/questions/29293394/where-does-the-1-symbol-come-from-when-using-llvms-libc, but I'm not sure why we're seeing this now. (Wonder if there has been a STL or compiler update.) Do you know anything about this? Can you and Kurt sync about this to make sure we handle all these cases correctly.

Reply:
Huh, I don't know anything about std::__1. @KurtCattiSchmidt , was that on windows/linux/mac? any special compile flags?

Reply:
__Cr comes from here: https://cs.chromium.org/chromium/src/build/config/c%2B%2B/BUILD.gn?type=cs&q=_LIBCPP_ABI_VERSION&sq=package:chromium&g=0&l=50

It looks like __1 is from here: https://cs.chromium.org/chromium/src/buildtools/third_party/libc%2B%2B/trunk/include/__config?type=cs&q=_LIBCPP_ABI_VERSION&sq=package:chromium&g=0&l=38

So it seems like we need to support __1 as well, for non-component builds. I think it'd be better to do this in a separate change because that also affects the other std::__Cr occurences in this file.

Reply:
OK, I made https://github.com/MicrosoftEdge/JsDbg/pull/109 and also updated this pull request to allow both __Cr and __1.

Reply:
Thank you, I'll take a look at your PR and see if it fixes my issue. I'm running Win10, with a debug component build.

Reply:
Looks like Windows was switched to libc++ in https://chromium.googlesource.com/chromium/src/+/4b298260f5327c09983083055a5775d5ce719cc1, so very recently.

You'll want to apply this pull request as well, in addition to #109

Reply:
Thanks @cbiesinger for finding the PR that changed this, and for fixing the stl helpers. I'll take a look at that PR shortly, but I'm good with fixing that separately.